ui/components/multichain/pages/send/components/content.tsx

```diff
@@ -1,20 +1,76 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { Box } from '../../../../component-library';
 import { getSendHexDataFeatureFlagState } from '../../../../../ducks/metamask/metamask';
-import { getSendAsset } from '../../../../../ducks/send';
+import {
+  GAS_INPUT_MODES,
+  getGasInputMode,
+  getGasLimit,
+  getGasPrice,
+  getIsBalanceInsufficient,
+  getMinimumGasLimitForSend,
+  getSendAsset,
+} from '../../../../../ducks/send';
+import AdvancedGasInputs from '../../../../app/advanced-gas-inputs';
+import { ConfirmGasDisplay } from '../../../../app/confirm-gas-display';
 import { AssetType } from '../../../../../../shared/constants/transaction';
-import { SendHexData } from '.';
+import { hexToDecimal } from '../../../../../../shared/modules/conversion.utils';
+import {
+  setCustomGasLimit,
+  setCustomGasPrice,
+} from '../../../../../ducks/gas/gas.duck';
+import { SendHexData, SendPageRow } from '.';
 
 export const SendPageContent = () => {
+  // Hex data
   const showHexDataFlag = useSelector(getSendHexDataFeatureFlagState);
   const asset = useSelector(getSendAsset);
-
   const showHexData =
     showHexDataFlag &&
     asset &&
     asset.type !== AssetType.token &&
     asset.type !== AssetType.NFT;
 
-  return <Box>{showHexData ? <SendHexData /> : null}</Box>;
+  // Gas data
+  const dispatch = useDispatch();
+  const gasPrice = useSelector(getGasPrice);
+  const gasLimit = useSelector(getGasLimit);
+  const minimumGasLimitForSend = useSelector(getMinimumGasLimitForSend);
+  const minimumGasLimit = hexToDecimal(minimumGasLimitForSend);
+
+  const gasInputMode = useSelector(getGasInputMode);
+  const insufficientBalance = useSelector(getIsBalanceInsufficient);
+
+  const updateGasPrice = (newGasPrice: string) => {
+    dispatch(updateGasPrice(newGasPrice));
+    dispatch(setCustomGasPrice(newGasPrice));
+  };
+
+  const updateGasLimit = (newLimit: string) => {
+    dispatch(updateGasLimit(newLimit));
+    dispatch(setCustomGasLimit(newLimit));
+  };
+
+  return (
+    <Box>
```

Comment:
Can we add a className or data-testId here?

Reply:
We shouldn't add those attributes until we're using them for something; i.e. tests